fasthtml/core.py

```diff
@@ -773,12 +773,13 @@ async def __call__(self, scope, receive, send) -> None:
 # %% ../nbs/api/00_core.ipynb
 class FtResponse:
     "Wrap an FT response with any Starlette `Response`"
-    def __init__(self, content, status_code:int=200, headers=None, cls=HTMLResponse, media_type:str|None=None):
+    def __init__(self, content, status_code:int=200, headers=None, cls=HTMLResponse, media_type:str|None=None, background: BackgroundTask | None = None):
         self.content,self.status_code,self.headers = content,status_code,headers
-        self.cls,self.media_type = cls,media_type
+        self.cls,self.media_type,self.background = cls,media_type,background
 
     def __response__(self, req):
         cts,httphdrs,tasks = _xt_cts(req, self.content)
+        tasks = getattr(self, 'background', tasks)
```

Comment:
I think `self.background` will *always* be set, so I'm not sure this is doing anything? Also IIUC it's the wrong way around - we should use the manually added `tasks` in the response if there, and only then fall back to `self.background`, shouldn't we?

Either way, it's a good idea to have tests or examples that exercise your code so that you know for sure whether it's working.

Reply:
Thanks for the feedback, we'll adjust the API accordingly and see about improving/adding to the tests and docs.